sound/soc/sof/ipc4-pcm.c

```diff
@@ -253,14 +253,14 @@ sof_ipc4_update_pipeline_state(struct snd_sof_dev *sdev, int state, int cmd,
  */
 
 static int sof_ipc4_chain_dma_trigger(struct snd_sof_dev *sdev,
```

Comment:
typo in commit message: :resenting -> resetting

Reply:
Oh, thanks!